pkg/message/message.go

```
@@ -31,12 +29,7 @@ func New(userMessage string, devMessage string, code sgerrors.ErrorCode, moreInf
 	}
 }
 func SendMessage(w http.ResponseWriter, msg Message, status int) {
-	data, err := json.Marshal(msg)
-	if err != nil {
-		logrus.Errorf("failed to marshall message: %v", err)
-		http.Error(w, "", http.StatusInternalServerError)
-		return
-	}
+	data, _ := json.Marshal(msg)
```

Please, check the error.
Other functions use similar flow, could it be moved to the message method?

---

Well, in general I agree, but it still strange for me why we check error all the time even for type that cannot fail in marshalling by design unless you add any unserializable type like chan

---

It's totally bad idea to ignore errors even if it looks like it can't happen.